dso-l2/src/test/java/com/tc/objectserver/core/impl/ManagementTopologyEventCollectorTest.java

```diff
@@ -163,7 +165,7 @@ public void testCreatePromoteDestroy() throws Exception {
     long consumerID = 1;
     // Note that we start in passive state.
     boolean isActive = false;
-    
```

Comment:
These whitespace changes clutter the PR.  Is it possible to move them to a separate commit or PR?

Reply:
I will remove this extra white spaces in this PR itself.